samples/near-realtime-clients/Core_8/ClientHub/Program.cs

```diff
@@ -15,41 +16,36 @@ static async Task Main()
         endpointConfiguration.UsePersistence<NonDurablePersistence>();
 
         endpointConfiguration.UseSerialization<SystemJsonSerializer>();
-        var routing = endpointConfiguration.UseTransport(new MsmqTransport());
-
-        routing.RegisterPublisher(
-            eventType: typeof(StockTick),
-            publisherEndpoint: "Samples.NearRealTimeClients.Publisher");
+        endpointConfiguration.UseTransport(new LearningTransport());
 
         endpointConfiguration.SendFailedMessagesTo("error");
 
         var conventions = endpointConfiguration.Conventions();
         conventions.DefiningEventsAs(type => type == typeof(StockTick));
 
-        endpointConfiguration.EnableInstallers();
+        var builder = WebApplication.CreateBuilder();
```

Comment:
Was it not possible to use NServiceBus.Extensions.Hosting with the `.UseNServiceBus(…)` extension method?

Reply:
Thanks for the tip.  It is indeed possible and I have made the changes